CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 1. Ensure you have [Node.js](https://nodejs.org/en/download/) installed, which comes bundled with [`npm`](https://github.com/npm/cli). Use it to install or upgrade [`yarn`](https://classic.yarnpkg.com/en/docs/install):
 
    ```sh
-   npm install --global yarn
+   npm install --global yarn lerna rimraf
```

Comment:
I think we should add lerna and rimraf as devDependencies rather than expect them to be global

Reply:
Yes, I can do that. Although then I have to execute the `lerna` and `rimraf` commands with `npx`. Do you want me to do that?

Reply:
@edwardfoyle I have removed the global deps.